cps-wallet-connector/README.md

```diff
@@ -0,0 +1,221 @@
+
+This problem statement is concerned with the issues surrounding Cardano's current and future wallet-web bridges.
+These interfaces are difficult to define and historically have been even harder to iterate upon.
+We wish to provide a comprehensive catalog of the current offerings; and their drawbacks to be able to make suggestions on future standards.
+
+### Acknowledgments
+
+<details>
+  <summary><strong>First workshop - 2023-11-27</strong></summary>
+
+  I would like to thank those that contributed to the first workshop hosted by Adam Dean and Ryan Williams ([see presentation slides with notes](https://docs.google.com/presentation/d/1lXw1yMf-Hyp1eVfaFnbTV9897r6pZEd6zpvLNMqT3cI/)).
+  - Beatrice Anihiri
+  - Denis Kalinin
+  - Evgenii Lisitskii
+  - George APEX Pool
+  - George Flerovsky
+  - George Humphreys
+  - Hernán Rajchert
+  - Jack Rousian
+  - Joshua Marchand
+  - Ken Fritschy
+  - Ken-Erik Ølmheim
+  - Leo H
+  - Marcel Baumberg
+  - Martynas Kazlaukas
+  - Michael Chappell
+  - Mircea Hasegan
+  - Nicolas Ayotte
+  - Rhys Bartels-Waller
+  - Robert Phair
+  - Rodolpho Ribeiro
+  - Steven Johnson
+  - Teodora Sevastru Lunn
+  - Thomas Lindseth
+  - Thomas Upfield
+  - Vladimir Kalnitsky
+
+</details>
+
+
+## Problem
+<!-- A more elaborate description of the problem and its context. This section should explain what motivates the writing of the CPS document. -->
+
+The core motivation of this document is to catalog the current state of Cardano's wallet connectors, discuss their flaws and identify keys concerns for future connector authors to be aware of.
+We hope that by discussing the issues we will inspire the next generation of Cardano wallet connectors so that the ecosystem can grow beyond it's first iterations.
+
+Ineffective dApp-connectors can cause a range of issues for wallets, dApp developers and thus users.
+Due to the nature of these connections there can often be unforeseen impacts of small design decisions.
+
+For users, connectors should be widely supported across platforms, secure and decentralized.
+For dApps, connectors should be expressive and extendable.
+For wallets, connectors should be secure, extendable and should not expect wallet's to go beyond their standard activities.
+
+### Core Concerns
+
+#### 1. Secure connection
+Security should remain of paramount concern within web 3.
+Connection standards must strive to above all not compromise the security of wallets or dApps.
+Connection standards should be aware of potential impacts of standard security vulnerabilities, such as man-in-middle attacks.
+Furthermore, standards should seek to be censorship resistant, by an ecosystem offering multiple connection standards we move in this direction.
+
+#### 2. Secure API
+The security of the API itself should again remain paramount.
+This means that no secret information should ever be allowed to leave the wallet.
+Furthermore if an operation requires the wallet to use a secret, the user should always be made aware. 
```

Comment:
Counterexample: [we are building a wallet extension for testers](https://cardano.ideascale.com/c/idea/103877), and one of the features is specifically the ability to skip confirmations (which is meaningful in the context of (semi-)automated testing).

We are not going to control how the wallet presents users its functionality by specifying the wallet connector standard. This is up to the wallet UI.

Reply:
> We are not going to control how the wallet presents users its functionality by specifying the wallet connector standard. 

I dont know, can we specify this? traditionally in CIP30 etc the connectors have specified this.
Perhaps a "wallet best practices" would be more suitable for this?